### PR TITLE
refactor: new project start page

### PR DESCRIPTION
**Pull request by topliceanurazvan**
## Changes

- Add `Sample Project` option
- Add `Your cluster live` option

## Screenshots

image

The git notice will be shown only if Monokle detects that git is not installed, the sample project text only if the users open the modal from the `Sample project` option

image

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by olensmar — APPROVED** [review]

File: src/components/organisms/NewProject/NewProject.tsx [hunk 1]
```diff
@@ -22,7 +32,33 @@ const NewProject: React.FC = () => {
...
+        'Not sure where to start? Explore a sample project containing everything you need to get started with just one click.',
+      itemAction: () => {
+        dispatch(openGitCloneModal({fromSampleProject: true}));
+      },
+    },
+    {
+      disabled: false,
+      itemId: 'cluster-live',
+      itemLogo: ClusterLive,
+      itemTitle: 'Your cluster live',
```
[the remark this answers is not in the document]
Comment (olensmar) [reply src/components/organisms/NewProject/NewProject.tsx:58]: or "Inspect your Cluster".. !? @chargio have any preference?

File: src/components/organisms/NewProject/NewProject.tsx [hunk 2]
```diff
@@ -22,7 +32,33 @@ const NewProject: React.FC = () => {
...
+  };
+
   const START_PROJECT_OPTIONS = [
+    {
+      disabled: false,
+      itemId: 'sample-project',
+      itemLogo: SampleProject,
+      itemTitle: 'Sample project',
+      itemDescription:
+        'Not sure where to start? Explore a sample project containing everything you need to get started with just one click.',
```
[the remark this answers is not in the document]
Comment (topliceanurazvan) [reply src/components/organisms/NewProject/NewProject.tsx:49]: image

I've added a link to the repo here

[the remark this answers is not in the document]
Comment (olensmar) [reply src/components/organisms/NewProject/NewProject.tsx:49]: nice!

**Review by olensmar — APPROVED** [review]
All good - only thing I noticed is that the "Back to Projects" button in the top left (when selecting "Explore your Cluster") always goes to "Recent Projects" - would be better if it went to where the user actually came from..

**Comment by topliceanurazvan** [comment]
> All good - only thing I noticed is that the "Back to Projects" button in the top left (when selecting "Explore your Cluster") always goes to "Recent Projects" - would be better if it went to where the user actually came from..

Should we maybe rename it to something like `Back to Start Page`, since being on settings and clicking on `Back to projects` looks a little bit confusing to me 🤔

**Comment by olensmar** [comment]
> > All good - only thing I noticed is that the "Back to Projects" button in the top left (when selecting "Explore your Cluster") always goes to "Recent Projects" - would be better if it went to where the user actually came from..
> 
> Should we maybe rename it to something like `Back to Start Page`, since being on settings and clicking on `Back to projects` looks a little bit confusing to me 🤔

sounds good - @julioramosest wdyt?

**Comment by julioramosest** [comment]
- agree with "Back to Start page". although this is a desktop app, so it does not really have pages. How does "Back to Start" sounds? or "Back to Start options"? or...?
- Let's have the link to the repo on the sample project modal popup in "GitHub sample repository" text label instead, better than on former one.

**Pull request merged**